velox/vector/fuzzer/tests/VectorFuzzerTest.cpp

```diff
@@ -499,6 +499,8 @@ TEST_F(VectorFuzzerTest, containerHasNulls) {
   opts.nullRatio = 0.5;
   opts.normalizeMapKeys = false;
   opts.containerHasNulls = true;
+  opts.allowDictionaryVector = false;
+  opts.allowConstantVector = false;
```

Comment:
The test is not well written and they assume the top level vector is not encoded in constant or dictionary.  I changed the random number generator to a faster implementation and these assumption no longer holds.  We can fix the test to decode instead of use `as<>` cast.